lms/lib/utils.py

```diff
@@ -0,0 +1,21 @@
+"""
+Helper methods for the LMS.
+"""
+
+
+def get_parent_unit(xblock):
+    """
+    Finds xblock's parent unit if it exists.
+
+    Returns:
+        xblock: Returns the parent unit xblock if it exists.
+        If no parent unit exists, returns None
+    """
+
+    while xblock:
+        parent = xblock.get_parent()
+        if parent is None:
+            return None
+        if parent.category == "sequential":
+            return xblock
+        xblock = parent
```

Comment:
I hate to keep harping on this somewhat innocent helper method, but it feels odd that "get_parent_unit" never checks that the xblock category is "vertical". How about making line 19

````
if xblock.category == "vertical" and parent.category == "sequential"